### PR TITLE
Fix errcheck issue in oauth.go

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28745

```release-note
NONE
```

### CODE REVIEW

**Comment by TheInvincibleRalph** [comment]
> @TheInvincibleRalph Thanks for the contribution. Looks like there are some more unchecked errors:
> 
> ```
> Error: channels/web/oauth.go:207:13: Error return value of `r.ParseForm` is not checked (errcheck) r.ParseForm()
> ```
> 
> Can you please take a look?

Done.

File: server/i18n/en.json [hunk 1]
```diff
@@ -10198,4 +10202,4 @@
     "id": "web.incoming_webhook.user.app_error",
     "translation": "Couldn't find the user."
   }
-]
+]
```
Comment (pvev) [inline server/i18n/en.json:10205]: please, add this line back

Comment (TheInvincibleRalph) [reply server/i18n/en.json]: Done, @pvev. Thank you!

Comment (TheInvincibleRalph) [reply server/i18n/en.json]: All checks are passed now.

**Review by BenCookie95 — APPROVED** [review]
LGTM, just need to fix the conflicts. Thanks 🚀

**Review by pvev — APPROVED** [review]
thanks for your contribution @TheInvincibleRalph 🚀

**Comment by TheInvincibleRalph** [comment]
> LGTM, just need to fix the conflicts. Thanks 🚀

Done

**Comment by TheInvincibleRalph** [comment]
@hanzei, this hasn't been merged.

**Comment by hanzei** [comment]
Let's merge :rocket:

**Pull request merged**